fmlloader/src/main/java/net/minecraftforge/fml/loading/targets/CommonClientLaunchHandler.java

```diff
@@ -40,7 +40,7 @@ public LocatedPaths getMinecraftPaths() {
         var lowcodelang = LibraryFinder.findPathForMaven(vers.forgeGroup(), "lowcodelanguage", "", "", vers.mcAndForgeVersion());
         var mclang = LibraryFinder.findPathForMaven(vers.forgeGroup(), "mclanguage", "", "", vers.mcAndForgeVersion());
 
-        return new LocatedPaths(mcstream.build().toList(), (a,b) -> true, modstream.build().toList(), List.of(fmlcore, javafmllang, lowcodelang, mclang));
```

Comment:
I considered that, however I decided against it as it would either imply everything else isn't nullable or mean that I'd have to comb through a much larger chunk of code to annotate stuff properly.